File: uri/salario/salario.lua
function foo( a, b )
    return a * b
end

local num, horas, valorhora, resultado
num = io.read("*n")
horas = io.read("*n")
valorhora = io.read("*n")
resultado = string.format("%.2f", foo(horas, valorhora))
print("NUMBER = "..num)
print("SALARY = U$ "..resultado)

File: uri/salario/salario.pas
program foo(Input,Output);

uses
    SysUtils;

function foo(var x: Integer; var y: Double) : Double;
begin
    foo := x * y;
end;

var 
    num, horas : Integer;
    valorhora, resultado : Double;

begin
    readLn(num);
    readLn(horas);
    readLn(valorhora);
    resultado := foo(horas, valorhora);
    writeln ('NUMBER = ',num);
    writeln ('SALARY = U$ ',Format('%.2f',[resultado]));
end.